vectorlm/utils/model_utils.py

```diff
@@ -142,7 +197,7 @@ def load_model_and_tokenizer(
 
 def fsdp_config(
     use_mp: bool,
-    layer_to_wrap: nn.Module,
+    model: nn.Module,
```

Comment:
I am not a fan of this change because it now requires the use of `fsdp_auto_wrap_policy` which creates a weird dependency with accelerate code (and its subsequent environment variables). I am in strong favor of reverting this change to what it was because it strictly uses PyTorch. See [here](https://github.com/meta-llama/llama-recipes/blob/c8f4bdac4107856e204b281e4fb611dc4e2de5dc/src/llama_recipes/utils/fsdp_utils.py#L6) for a better implementation of what you might be trying to get at. We should stick to using Torch FSDP wrappers.

Reply:
Resolved in c1b31c4. Also eliminated implicit dependency on `accelerate` through `peft.utils.other.fsdp_auto_wrap_policy`. Thanks for the suggestions.